src/main/java/Duke.java

```diff
@@ -1,10 +1,28 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n");
+        Scanner sc = new Scanner(System.in);
+        String input = "";
+
+        while (true) {
+            input = sc.nextLine();
+            if (input.equals("bye")) {
+                break;
+            }
+            System.out.println("____________________________________________________________");
```

Comment:
Instead of calling this command every time, may I suggest having a separate variable to store the line break and a separate method to print the line break